### DISCUSSION TITLE
Display Logo in Custom Command

### DISCUSSION THREAD

**Discussion by EsoCoding** [laravel-zero/laravel-zero#413] [General]
I wanted to display the logo module into my custom command. I did found out how, but maybe it would be nice to explain it into the documentation of the logo module? In case anyone wonders... I'll add it here:

`new FigletString(config('app.name'), config('logo', []));`

**Comment by owenvoke** [comment]
Feel free to open a PR on the docs to add this. 👍🏻

**Comment by EsoCoding** [comment]
Considered it done, tough probably it needs some spell checking ;)
